boot/bootutil/include/bootutil/bootutil.h

```diff
@@ -57,6 +60,17 @@ struct boot_rsp {
     uint32_t br_image_off;
 };
 
+/* This is not actually used by mcuboot's code but can be used by apps
```

Comment:
Shouldn't there be an ioctl() or something for this? What if the format changes in v2?

In general, I think we should consider how to share code between the bootloader and applications a bit. Putting this into an existing header alongside code that applications should not use seems like asking for trouble, a bit. Could we start by having a header which is only for shared definitions?